libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -371,11 +385,22 @@ public void onImeBack() {
 
     @Override
     public void setTitle(CharSequence text) {
+        if (title == null) {
```

Comment:
Should we use `if (title == null && text !=null)` here (and similarly for `setContent`)?

Reply:
Been thinking a bit on this - decided to make sure that if `text` is null we set it to an empty string and continue, as it's perfectly possible that we'd want to empty then contents, and also it'd be aligned with what `getContent()` and `getTitle()` do (that is, they both return empty strings if there's no content)
addressed as explained in adb10a6

Reply:
The reason I mentioned adding a null check for `text` is that `setTitle()` has this:
```
if (title == null) {
   mLateInit = true;
   mLateInitTitle = text;
   return;
}
```
So if `text` is null then `title` will always be null and `mLateInit` will always be set to true. Similarly for `setContent()`.

Reply:
Bah, ignore that last comment - I was misreading the code. Sorry!